.env.rc1

```diff
@@ -60,7 +60,7 @@ CELOCLI_STANDALONE_IMAGE_TAG="0.0.42"
 MOCK_ORACLE_CRON_SCHEDULE="*/5 * * * *"
 
 ORACLE_DOCKER_IMAGE_REPOSITORY="celoprod.azurecr.io/oracle"
-ORACLE_DOCKER_IMAGE_TAG="3cc6459db821f03ce0df12d2891b3ff4b34e3f05"
+ORACLE_DOCKER_IMAGE_TAG="975d290fce7098a7e6026edd964835ec495026ef"
```

Comment:
I usually try to not update mainnet until tested for security

Reply:
Resolving, as it's deployed to mainnet already.